sdk/src/types/block/rand/parents.rs

```diff
@@ -7,6 +7,9 @@ use crate::types::block::{
 };
 
 /// Generates random strong parents.
-pub fn rand_strong_parents<const MAX: u8>() -> Parents<1, MAX> {
-    Parents::from_vec(rand_block_ids(rand_number_range(Parents::<1, MAX>::COUNT_RANGE).into())).unwrap()
+pub fn rand_parents<const MIN: u8, const MAX: u8>() -> Parents<MIN, MAX> {
```

Comment:
Nice, I almost commented in the previous PR that I didn't like it and ended up not doing it

Reply:
this comment made me notice the comment above is outdated, thanks lol